Training/Training/change.c

```diff
@@ -0,0 +1,70 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// change.c
+// Program to perform and test arithmetic operations on complex numbers.
+// ------------------------------------------------------------------------------------------------
+
+#include<stdio.h>
+#include<errno.h>
+#include<stdbool.h>
+#include<stdlib.h>
+#include<string.h> // USED
+
+void OptimalChangeCalculator (int* coins, int rem, int* denominations) {
+   for (int i = 0; i < 4; i++) {
+      coins[i] = rem / denominations[i];
+      rem %= denominations[i];
+   }
+}
+
+static void Testcases (int* denominations) {
+   int input[] = { 95,75,67,79 }, expected[][4] = { 9,1,0,0,7,1,0,0,6,1,1,0,7,1,2,0 }, output[4];
```

Comment:
```suggestion
   int input[] = { 95,75,67,79 }, expected[][4] = { { 9, 1, 0, 0 }, { 7, 1, 0, 0 }, { 6, 1, 1, 0 }, {7, 1, 2, 0 }, output[4];
```
2D arrays should be declared as such for readability.